VstsTfsSample/README.md

```diff
@@ -0,0 +1,46 @@
+# VSTS TFS Client Sample
+
+For windows native applications which want to target both VSTS and TFS we recommend using [Azure Active Directory Authentication Library (ADAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-libraries) to generate interactive sign in prompts for VSTS users and [Client Libraries for VSTS and TFS](https://www.visualstudio.com/en-us/docs/integrate/get-started/client-libraries/dotnet) to leverage Windows credential authentication for TFS users.
```

Comment:
Why wouldn't we recommend client for both?  Wouldn't that make the experience simpler?